xml/hwbp_registers.xml

```diff
@@ -223,15 +223,16 @@
         hold all valid virtual addresses but it need not be capable of holding
         other values.
 
-        If the A extension is supported, then trigger behavior is as follows
-        for the load and store bits:
-        \vspace{-0.2in}
-        \begin{tightlist}
+        \begin{steps}{If the A extension is supported, then trigger behavior is as follows
+        for the load and store bits:}
         \item {\tt lr} instructions are loads
         \item successful {\tt sc} instructions are stores
         \item it is \unspecified\ whether failing {\tt sc} instructions are stores or not
-        \item Each AMO instruction is both a load and a store
-        \end{tightlist}
+        \item Ideally each AMO instruction is a load for the read portion of
+        the operation and a store for the write part of the operation.
+        Implementations may choose to implement only the load part, only the
+        store part, or neither.
+        \end{steps}
```

Comment:
Is "ideally" the right word here?  Maybe "nominally"?  I would also qualify the implementation's choices with something like "_In certain cases,_ implementations may choose..." since it is not allowed to remove the load or store in most cases.

Reply:
Is it useful to allow implementations to not implement:
- load address triggers on AMOs
- store address triggers on AMOs
- load value triggers on AMOs where the value is actually written to the register file (so the hart knows the value)

Wouldn't any hart (no matter how it's implemented) already have all of the information for all of those?

Note that the later paragraph already says that it's an implementation option whether to implement load value triggers on AMOs where the value is not written (for fire-and-forget AMO operations) and whether to implement store value triggers at all (for far atomics).

Reply:
Rereading what Paul wrote originally, I think my PR isn't intended to actually change anything.

The text below makes slightly more sense to me, but I don't think it changes anything. Thoughts?

* Each AMO instruction is a load for the read portion of the operation. The address is always available to trigger on, although the value loaded might not be, depending on the hardware implementation.
* Each AMO instruction is a store for the read portion of the operation. The address is always available to trigger on, although the value stored might not be, depending on the hardware implementation.

Reply:
That seems good to me!

Reply:
Except that "read" should be "write" on the second item, that looks OK to me.